src/tactic/rcases.lean

```diff
@@ -945,5 +945,36 @@ add_tactic_doc
   decl_names := [`tactic.interactive.obtain],
   tags       := ["induction"] }
 
+/--
+The `rsuffices` tactic is an alternative version of `suffices`, that allows the usage
+of any syntax that would be valid in an `obtain` block. This tactic just calls `obtain`
+on the expression, and then `rotate 1`.
+-/
+meta def rsuffices (h : parse obtain_parse) : tactic unit :=
+focus1 $ obtain h >> tactic.rotate 1
+
+add_tactic_doc
+{ name       := "rsuffices",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.rsuffices],
+  tags       := ["induction"] }
+
+/--
+The `rsufficesI` tactic is an instance-cache aware version of `rsuffices`; it resets the instance
+cache on the resulting goals.
+-/
+
+-- TODO: this (I think) resets the instance cache also on the goal you need to make the instance.
+-- This is not necessary; do we need this?
```

Comment:
I think it's fine, `resetI` is already used in cases where something subtler would suffice. So I'll leave it up to you whether you want to keep this comment for future improvement.

Reply:
I actually saw a case where this would be wanted, and I think that's definitely a good enough reason to keep it around. I'll remove the comments!